gateway/webhook/setup.go

```diff
@@ -55,6 +56,14 @@ func Setup(gwHandle Gateway, transformerFeaturesService transformer.FeaturesServ
 	maxTransformerProcess := config.GetIntVar(64, 1, "Gateway.webhook.maxTransformerProcess")
 	// Parse all query params from sources mentioned in this list
 	webhook.config.sourceListForParsingParams = config.GetStringSliceVar([]string{"Shopify", "adjust"}, "Gateway.webhook.sourceListForParsingParams")
+
+	webhook.config.forwardGetRequestForSrcMap = lo.SliceToMap(
+		config.GetStringSliceVar([]string{"adjust"}, "Gateway.webhook.forwardGetRequestForSrcs"),
```

Comment:
I would avoid using a config for this. Similar for `sourceListForParsingParams`.

As configuration skew across environments can cause issues.

Reply:
Can you explain how and why you think using config would cause issues ?

Reply:
This configuration contains source-specific business logic.

* Configuration can diverge in different environments, having a source working in one, but not in the other.
* Configuration circumvents testing and usual release processes.


IMO `Gateway.webhook.forwardGetRequestForSrcs` `Gateway.webhook.sourceListForParsingParams` should be converted to constants any change on them should require a server release and source-specific tests.

We can also consider moving this logic to rudder-transformers. rudder-server or ingestion-svc will pass all the methods (GET, POST, PUT .. ) and `rudder-transfomer` will have logic to reject them or accept them.